packages/lean-symbols/1.0.0/abbreviations.json

```diff
@@ -0,0 +1,1836 @@
+{
```

Comment:
Do we need this file, as you have provided a link to the source in your README.md (thank you)?

Reply:
We don't strictly need it. I put in, in case the file in the VScode extension is updated and becomes incompatible, but I could of cause also just point to a specific commit. I think I will do that instead.

Reply:
Ah it already does this actually.

Reply:
I have made it such that the file is fetched now and deleted it from the repo.